charts/images.yaml

```diff
@@ -233,21 +219,7 @@ images:
 - name: csi-resizer
   sourceRepository: github.com/kubernetes-csi/external-resizer
   repository: registry.k8s.io/sig-storage/csi-resizer
-  tag: "v1.5.0"
```

Comment:
You need to keep the CVE categorizations

Reply:
The removed CVE categorizations belongs to the csi-snapshotter v3.0.3.
The visualisation of the diff is very confusing here, but the CVE cats of the csi-resizer are unchanged.